doc/ADR.md

```diff
@@ -0,0 +1,22 @@
+# ADR - Architectural Decision Records
+
+This is a lightweight documentation system meant to record
+high-level decisions taken during the design of the project.
+See <https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions>
+for the inspiration.
+
+ADRs are typically used on larger projects. We include them here
+to keep track of foundational decisions which affect contributors
+over the longer term.  ADRs start as a [new issue](https://github.com/hytest-org/hytest/issues/new) tagged with <kbd>ADR</kbd>.  Discussion and decisions are included
+in the issue's discussion thread. They are added to this index once accepted as a
+convenience to see them all in one place. (also available by
+a [search of the issues in this repo](https://github.com/hytest-org/hytest/issues?q=label%3AADR).
+
+| ADR Issue No.| Status | Description |
```

Comment:
I'm not sure I understand what this table is? It is just showing a few examples of our ADRs?

Reply:
The original idea was to have a flat file for somebody to read... with a link to every accepted ADR in the issue database.   You could argue that it is redundant, in that the issue search gives more or less the same thing.

Reply:
Ok - I'm fine with this! I was confused by the number scheme - are those just the issue numbers?

Reply:
yes... issue numbers.  Now that you mention it...that is a little odd.  Perhaps I should add 'issue' in front of each.  or some such.

Reply:
Maybe this is fine...I was confused because I thought we were transferring over your ADRs from [your current repo](https://github.com/USGS-python/hytest_notebook_tutorials/tree/dev/doc/adr), and the numbers weren't aligning. But I see that you are suggesting that we just keep this markdown file up to date and link to the issues that are resolved with a decision (and do not copy over the current ADRs.